conferences.html

```diff
@@ -161,9 +161,9 @@ <h4>Start a local community</h4>
                 </div>
         </div>
 
-        <script src="vendor/jquery.min.js"></script>
-        <script src="vendor/materialize.min.js"></script>
-        <script src='./conferences-page.js'></script>
+        <script src="/vendor/jquery.min.js"></script>
+        <script src="/vendor/materialize.min.js"></script>
+        <script type="module" src="/src/conferences-page.js"></script>
 </body>
```

Comment:
The script produced by vite needs the type to be module and also the path changes